src/set_theory/game.lean

```diff
@@ -45,9 +45,9 @@ quotient.lift₂ (λ x y, x ≤ y) (λ x₁ y₁ x₂ y₂ hx hy, propext (le_co
 instance : has_le game :=
 { le := le }
 
-@[refl] theorem le_refl : ∀ x : game, x ≤ x :=
+theorem le_refl : ∀ x : game, x ≤ x :=
 by { rintro ⟨x⟩, apply pgame.le_refl }
-@[trans] theorem le_trans : ∀ x y z : game, x ≤ y → y ≤ z → x ≤ z :=
+theorem le_trans : ∀ x y z : game, x ≤ y → y ≤ z → x ≤ z :=
```

Comment:
Does the linter reject these now?

Reply:
The issue is not with the linter. There are type check an actual error in the `calc` mode proofs and `refl` proofs. Lean tries to use `game.le_trans` where it should be using `pgame.le_trans`.
Similarly, for `refl`.

Reply:
There's an error here: https://github.com/leanprover-community/mathlib/blob/41674ba8dd5a79d62d0024995e6156e03de6f955/src/set_theory/surreal.lean#L389

and one here: 
https://github.com/leanprover-community/mathlib/blob/41674ba8dd5a79d62d0024995e6156e03de6f955/src/set_theory/surreal.lean#L531

Reply:
This worries me quite a bit - I don't see why lean at any point thinks these two are interchangeable
```lean
theorem game.le_trans (x y z : game) : x ≤ y → y ≤ z → x ≤ z
theorem pgame.le_trans {x y z : pgame} : x ≤ y → y ≤ z → x ≤ z
```

Reply:
Ah, here's the cause: https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/norm_num.20can't.20solve.201.20.E2.89.A4.202/near/226015763

Reply:
Thanks for finding this! This thread is unresolved though :(
Is there any workaround for this?

Reply:
I wonder if there is some way to assign "priorities" to `trans` and `refl`...

Reply:
Priorities wouldn't help, theproblem is that the `refl` tactic says "find me the refl lemma for `has_le.le`", not "find me the refl lemma for `@has_le.le pgame`".

Your choice to remove them was sensible here - but I'd recommend perhaps going further and either in this PR or a follow-up, removing all the refl and trans attributes about le and lt, and adding them back locally using `local attribute [trans] pgame.le_trans` etc.

Reply:
I tried this, but it breaks all the `calc` proofs (and there are lots of them) in surreal.lean

Reply:
Several `calc` proofs are about `numeric` so can't be moved to pgame.lean